### PR TITLE
working-groups: add dm HA working group

### PR DESCRIPTION
**Pull request by nolouch**
add dm HA working group.
Signed-off-by: nolouch <nolouch@gmail.com>

### CODE REVIEW

File: working-groups/wg-dm-ha.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# DM HA Working Group
+
+## Goals
```
Comment (zz-jason) [inline working-groups/wg-dm-ha.md]: please also update the `working-groups/README.md` file.

File: working-groups/wg-dm-ha.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Currently,  DM cluster has only one DM-master process. When the DM-master process is abnormal, task management, cluster management, and shard DDL coordination can no longer be performed. Also, the current set of DM cluster has only one DM-worker process for each upstream MySQL/MariaDB instance, when the DM-worker process is abnormal, data migration related to the upstream MySQL/MariaDB instance is interrupted. In addition, for some metadata (task configuration, cluster version, etc.), there is only one copy of data locally on the DM-worker node. When the local data copy is damaged, the data migration task is difficult to recover. We need to improve the high-availability and usability of DM to solve the actual pain points of users.
+* Make DM become HA, which includes making it tolerant to process crashes, machine failures, network partitioning and more.
+* Combine DM-master into DM-worker as one binary file.
+
+## Work and Meeting
+
+* Regular Working Group Meeting: 4pm on every Tuesday
```
Comment (zz-jason) [inline working-groups/wg-dm-ha.md]: please add the time zone info.

**Comment by zz-jason** [comment]
@nolouch please resolve the conflict.

File: working-groups/wg-dm-ha.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# DM HA Working Group
+
+## Goals
+Currently,  DM cluster has only one DM-master process. When the DM-master process is abnormal, task management, cluster management, and shard DDL coordination can no longer be performed. Also, the current set of DM cluster has only one DM-worker process for each upstream MySQL/MariaDB instance, when the DM-worker process is abnormal, data migration related to the upstream MySQL/MariaDB instance is interrupted. In addition, for some metadata (task configuration, cluster version, etc.), there is only one copy of data locally on the DM-worker node. When the local data copy is damaged, the data migration task is difficult to recover. We need to improve the high-availability and usability of DM to solve the actual pain points of users.
+* Make DM become HA, which includes making it tolerant to process crashes, machine failures, network partitioning and more.
+* Combine DM-master into DM-worker as one binary file.
+
+## Work and Meeting
+
+* Regular Working Group Meeting: 4pm on every Tuesday at GMT+8 (biweekly).
```
Comment (gingerkidney) [inline working-groups/wg-dm-ha.md]: please add time zone, the rest LGTM

**Comment by nolouch** [comment]
@zz-jason  @gingerkidney  PTAL

**Pull request merged**